### PR TITLE
Refine typography. #287.

### LINKED ISSUE #287: Refine typography
See [clarification](https://github.com/clevercanary/data-browser/issues/277).

#### Font Family
* Default is Inter.
* HCA/LungMAP requires overrides with DIN.
~~* AnVIL requires overrides with Open Sans.~~
* Update Figma to include DIN fonts.

#### Color
* Fallback for all is `ink`.

### PR DESCRIPTION
**Pull request by frano-m**
### Ticket
Closes #287 .

### Reviewers
@MillenniumFalconMechanic .

### Changes
- Updated theme with responsive typography and any site specific typography overrides.
- Modified theme from a constant to a function with parameter `customTheme`.
- Removed any unnecessary reference to colour 'ink' in "touched" components.
- Added Din font.

### Outstanding
- To be able to "switch" between site configs for SB.

### CODE REVIEW

File: explorer/.storybook/preview.js [hunk 1]
```diff
@@ -23,6 +24,8 @@ export const parameters = {
 };
 
 const withThemeProvider = (Story, context) => {
+  const currentConfig = useConfig();
+  const theme = getAppTheme(currentConfig.theme);
```
Comment (frano-m) [inline explorer/.storybook/preview.js:28]: By using the useConfig hook and passing this in to `getAppTheme` we are able to access any custom theme defined in `config` for SB, without error.

File: explorer/pages/_document.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ class MyDocument extends Document {
     return (
       <Html>
         <Head>
+          <link rel="stylesheet" href="https://use.typekit.net/qhb0geh.css" />
```
Comment (MillenniumFalconMechanic) [inline explorer/pages/_document.tsx:8]: Is this loading DIN for every site config?

**Review by MillenniumFalconMechanic — COMMENTED** [review]
Thanks Fran! Just one q for when you have a moment.

File: explorer/pages/_document.tsx (hunk 2 above)
Comment (MillenniumFalconMechanic) [reply explorer/pages/_document.tsx:8]: As per our conversation, let's address this in a separate ticket.

**Comment by frano-m** [comment]
@MillenniumFalconMechanic new fonts downloaded for both HCA and AnVIL build and the typography overrides for HCA work as expected.

**Review by MillenniumFalconMechanic — APPROVED** [review]
Thanks F!

**Pull request merged**